vendor/vendor.json

```diff
@@ -613,6 +613,12 @@
 			"revision": "e214231b295a8ea9479f11b70b35d5acf3556d9b",
 			"revisionTime": "2018-01-08T23:09:05Z"
 		},
+		{
+			"checksumSHA1": "1txgvJO9bgE703kK2EQ/NATy1Rg=",
+			"path": "github.com/coreos/go-systemd/dbus",
```

Comment:
There is only one dependency? I checked the file and found that there are a lot of unless and non-go files in the vendor. Could you mind to provide your govendor command when you add the dependency? Thanks

Reply:
of course, what I usually do when add new dependencies :
```
go get package
govendor add/update package
```

Reply:
I afraid that it doesn't work. `go get` will fetch the latest code in your gopath. I think we should use `govendor fetch package@version`